### ISSUE TITLE
Regarding the issue of being unable to reproduce test results

### ISSUE THREAD

**Issue by ClimBin** [rccchoudhury/rlt#12]
Dear Authors,

I greatly appreciate your work on RLT. Recently, I downloaded the pre-trained checkpoint (ViT-B) for Kinetics-400 from your repository to perform inference. However, I obtained results that are lower than those reported in your paper.

Here are the details of my setup:

Dataset: Kinetics-400 downloaded from the official website
Data preprocessing: Removed ~14 corrupted videos, leaving 38,671 videos in the test set
Hardware: Single A100 GPU
Procedure: Followed your provided instructions exactly
I would appreciate any guidance on what might cause this discrepancy.

<img width="1056" height="117" alt="Image" src="https://github.com/user-attachments/assets/952e53f3-0669-43a7-a537-7001485c599e" />

Best regards.
